.github/workflows/nightly.yml

```diff
@@ -116,6 +116,7 @@ jobs:
             ${{ matrix.run_tests_parameters }}
             -d zend_extension=opcache.so
             -d opcache.enable_cli=1
+            -d opcache.jit=disable
```

Comment:
Instead of making opcache.jit opt-out, can you make it opt-in? This may require an input to the actions. See `runTestsParameters` on how that is done. Gotcha for bools: They are coerced to strings, to you will need something like `${{ inputs.withJit == 'true' && '...' || '' }}`. Also note the explicit `|| ''`. Without it, you'll get `false` in the output. GitHub actions is *something*.